settings.gradle.kts

```diff
@@ -2,7 +2,6 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
-
 plugins {
     id("org.gradle.toolchains.foojay-resolver-convention") version "0.9.0"
```

Comment:
I think you can move this plugin to the build gradle file

Reply:
This does not work. ```
An exception occurred applying plugin request [id: 'org.gradle.toolchains.foojay-resolver-convention', version: '0.9.0']
> Failed to apply plugin 'org.gradle.toolchains.foojay-resolver-convention'.
   > Settings plugins must be applied in the settings script.
```